frontend/src/utils/metamask.ts

```diff
@@ -0,0 +1,26 @@
+const walletAddToken = async (
+  tokenAddress: string,
+  tokenSymbol: string,
+  tokenImgUrl: string,
+  tokenDecimals: number
+) => {
+  try {
+    // wasAdded is a boolean. Like any RPC method, an error may be thrown.
+    const wasAdded = await window.ethereum.request({
+      method: 'wallet_watchAsset',
+      params: {
+        type: 'ERC20', // Initially only supports ERC20, but eventually more!
+        options: {
+          address: tokenAddress, // The address that the token is at.
+          symbol: tokenSymbol, // A ticker symbol or shorthand, up to 5 chars.
+          decimals: tokenDecimals, // The number of decimals in the token
+          image: tokenImgUrl, // A string url of the token logo
+        },
+      },
+    });
```

Comment:
Please remove wasAdded.

You can find these types of errors with `yarn run lint`
```
src/utils/metamask.ts
--
10:46:26.446 | Line 9:11:  'wasAdded' is assigned a value but never used  @typescript-eslint/no-unused-vars
```

Reply:
hmmm this variable can be useful in an if else statement if needed to check if the token was correctly added tho.

Reply:
but I'll still remove it and leave a comment there.

Reply:
fixed